services/src/messaging/src/main/java/org/openkilda/messaging/Message.java

```diff
@@ -99,8 +99,7 @@ public Message(@JsonProperty(TIMESTAMP) final long timestamp,
      * @param timestamp     message timestamp
      * @param correlationId message correlation id
      */
-    public Message(@JsonProperty(TIMESTAMP) final long timestamp,
-                   @JsonProperty(CORRELATION_ID) final String correlationId) {
+    public Message(final long timestamp, final String correlationId) {
```

Comment:
Why remove @JsonProperty?

Reply:
We have already created constructor which is responsible for the creation objects from json. Also we mark only one constructor with @JsonCreator and it parameters with @JsonPropertyr.
https://stackoverflow.com/questions/15931082/how-to-deserialize-a-class-with-overloaded-constructors-using-jsoncreator
This constructor we use just to create object from javacode (not deserializing), so we don't need it marked with @JsonCreator